### DISCUSSION TITLE
How do I trigger my workflow when a comment on a PR code is created?

### DISCUSSION THREAD

**Discussion by ldeluigi** [community/community#39097] [Actions] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/44567586/201734612-b00216c2-acbb-4539-b978-8015e6149627.png)
like the one you see in the picture.

I can't find an event corresponding to those.

**Comment by david-engelmann** [comment]
[Here is the documentation on pull request comments](https://docs.github.com/en/rest/pulls/comments). I'm able to see cases where I start a review

**Reply by ldeluigi** [reply]
I need to trigger my github actions workflow with them

**Comment by david-engelmann** [comment]
[There are several actions that accomplish, here's one of them](https://github.com/Khan/pull-request-comment-trigger). It could possibly be used in a workaround

**Comment by laughedelic** [comment, marked as answer]
You should consult the docs for [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows). There you can find [this](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_comment-use-issue_comment):

> ### `pull_request_comment` (use `issue_comment`)
>
> To run your workflow when a comment on a pull request (not on a pull request's diff) is created, edited, or deleted, use the [`issue_comment`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issue_comment) event. For activity related to pull request reviews or pull request review comments, use the [`pull_request_review`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review) or [`pull_request_review_comment`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review_comment) events.
